File: src/main/java/com/hyperdondon/blissgemsremake/api/util/TimeUtils.java
package com.hyperdondon.blissgemsremake.api.util;

public final class TimeUtils {
    public static long FromMinutesAndSeconds(int min, int sec) {
        for (int i = 0; i < min; i++)
            sec += 60;
        return FromSeconds(sec);
    }

    public static long FromSeconds(int sec) {
        long milli = 0;
        for (int i = 0; i < sec; i++)
            milli += 1000;
        return milli;
    }

    public static int ToSeconds(int milli) {
        int sec = 0;
        while (milli > 0) {
            milli -= 1000;
            sec++;
        }
        return sec;
    }
}

File: src/main/kotlin/com/hyperdondon/blissgemsremake/internal/util/TimeUtils.kt
package com.hyperdondon.blissgemsremake.internal.util

object TimeUtils {
    fun fromMinutesAndSeconds(min: Int, sec: Int): Long {
        var sec = sec
        for (i in 0..<min) sec += 60
        return fromSeconds(sec)
    }

    fun fromSeconds(sec: Int): Long {
        var milli: Long = 0
        for (i in 0..<sec) milli += 1000
        return milli
    }

    fun toSeconds(milli: Int): Int {
        var milli = milli
        var sec = 0
        while (milli > 0) {
            milli -= 1000
            sec++
        }
        return sec
    }
}
